src/server/api/routers/example.ts

```diff
@@ -15,7 +15,7 @@ export const exampleRouter = createTRPCRouter({
     }),
 
   getAll: publicProcedure.query(({ ctx }) => {
-    return ctx.prisma.example.findMany();
+    return ctx.prisma.user.findMany();
```

Comment:
just changed this to the user instead of the example as I deleted the example table but this might be risky